worker/src/queues/ses/helpers/FileService.ts

```diff
@@ -16,6 +31,7 @@ export default class FileService {
    */
   async getFile(url: string): Promise<{ contentType: string; data: Buffer }> {
     try {
+      this.validateFileLocation(url);
```

Comment:
```suggestion
  async getFile(url: string): Promise<{ contentType: string; data: Buffer }> {
    const isValid = this.validateFileLocation(url);
    if(!isValid) {
      throw new ApplicationError("FILE", "ORIGIN_NOT_ALLOWED", `The specified file location ${url} is forbidden`);
    }


```

this with the other suggestion will prevent having to keep throwing/catching and rethrowing.